### PR TITLE
Alumni checkbox will only be set checked by default if the user is only an alumni

### PR DESCRIPTION
**Pull request by antoniavonto**
Alumni checkbox will only be set checked by default if the user is only an alumni.
Taking in consideration that there are people that fall into many categories (seniors for example), we discussed what would be the best idea for the default.

### CODE REVIEW

**Review by amos-cha — APPROVED** [review]
LGTM, don't merge until after release

**Review by russtuck — APPROVED** [review]
LGTM

**Pull request merged**
